src/components/Slider/Slider.js

```diff
@@ -22,6 +22,11 @@ class Slider extends React.Component {
   }
 
   render() {
+    // There's no need to try to render the Slider, it will not be usable
+    if (this.props.range.min === this.props.range.max) {
```

Comment:
Yes, the small issue I have here is that basically we could extend this to all the other components, always with the `shouldAutoHideContainer` condition. What about renaming it, since it's internal, to a `containerEmpty` condition, create an `EmptyContainer` decorator that would return `null` or `<div/>` as `AutoHide` but only for the content, and make both of them use the same condition?